src/cmd-build

```diff
@@ -298,7 +298,7 @@ if [ ! -f /lib/coreos-assembler/.clean ]; then
 fi
 
 # And create the ostree repo tarball containing the commit
-ostree_tarfile_path=${name}-${buildid}-ostree.tar
+ostree_tarfile_path=${name}-${buildid}-ostree-${basearch}.tar
```

Comment:
Why not `.${basearch}` here to be consistent?

Reply:
that was a mistake - fixed